LICENSES/LicenseRef-ReplaceMe.txt

```diff
@@ -0,0 +1 @@
+Unlicense.txt
```

Comment:
How does it work? Why is it done this way?

Reply:
That’s a symlink.